UC-Berkeley-Extension/global
# Layout builder landing page

## Description
- This PR turns on Layout Builder module, along with Layout Builder Restrictions.
- Creates a separate landing page (landing page LB) to be used with Layout Builder.
- Creates placeholder hero custom block type (and template) that will need adjusting once the Hero component is completed.
- Adds code that allows you to add classes when creating/configuring a LB section.
- Adds code for block template suggestions

## Testing instructions
- `ddev start`, `ddev ssh`, `drush cim`, `drush cr`
- Create a new Layout Builder LB content page
- Go to Layout tab, start adding sections and blocks to test.
- Add section -> Add one column -> Add block -> Create custom block -> Hero
<img width="306" alt="Screen Shot 2020-10-27 at 5 05 33 PM" src="https://user-images.githubusercontent.com/10488517/97367173-a649e400-1876-11eb-8b60-f206767280e0.png">

- Make sure the Hero block is added properly. This block will change once Hero component is done.

## Code Review Notes
- I merged in main nav PR (#35 ) in here.
